examples/globe.js

```diff
@@ -48,17 +45,19 @@ if (!renderer) {
     };
 
     // Add one imagery layer to the miniview
-    itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(function _(layer) { miniView.addLayer(layer); });
+    itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(function _(layer) {
```

Comment:
why not the following ?
` itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(miniView.baseLayer.addColorLayer); `

Reply:
You're right. I pushed a fix, thanks.